### DISCUSSION TITLE
Ayuda con pokedex

### DISCUSSION THREAD

**Discussion by FernandoRizo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#547] [Q&A]
Hola! sé que es un poco tarde pero a penas he tenido tiempo para tratar de ponerme al corriente. Tengo duda en saber cómo muestras el resto de atributos del pokemon, no entiendo cómo poner código en js para que muestre, entiendo cómo está estructurado el código del MC para que muestre la imágen pero al momento de querer mostrar lo demás no hayo la forma para que en html pueda poner el resto. Si podrían ayudarme un poco, espero haberme explicado. 
He puesto una variable para las habilidades y mostrar los movimientos pero no sé en qué parte de html poner para que los muestre :c 

![imagen_2022-03-27_200534](https://user-images.githubusercontent.com/99287660/160314332-d6bf1c45-5d60-40b5-ab76-1eeef1de2e55.png)

**Comment by Charlychavee** [comment]
Te comparto una foto de mi código a ver si te puede servir de algo
![image](https://user-images.githubusercontent.com/33816014/160319222-7a507149-1cee-4b68-a610-ad2b0f68721e.png)

**Comment by enrique3240** [comment]
Lo que sucede son varias cosas.

1) Cuando obtienes la información de la API, y la colocas en tu variable `pokeH`, ya le estás asignando el valor de las habilidades (p. ej,: overgrow,chlorophyll), ya no estás obteniendo una URL para enviarla a tu función 'pokeHability = (url)' (de hecho no se requiere URL).

2) En tu función `pokeHabiliity = (url)` obtienes correctamente el elemento con la id `"pokeName"` , pero no se puede asignar el src de ese elemento, ya que no es el adecuado. Sugiero lo siguiente:

Ya obtuviste el valor de las habilidades como te comenté en el 1), así que puedes enviar directamente esa información a tu función. En tu función de habilidades, cambia el nombre de la variable `url` a por ejemplo `abilities` o `habilidades` para que no te confundas. De ahí, dependiendo del elemento en el que lo muestres en el HTML. Te anexo código de ejemplo.

![code](https://user-images.githubusercontent.com/17033541/160345673-fcadf7a2-6829-4cb9-8fc4-5da096576a58.jpg)

Con eso ya se debería mostrar la información en tu página. Saludos.

**Reply by FernandoRizo** [reply]
@enrique3240 Muchas gracias! Si, ya me mostraba los movimientos en la consola, mi problema es, cómo ligar js a html para que muestre los movimientos en la página así como se ligó para la imágen, según yo ya se debería de mostrar por el fetch pero no lo hace. No sé si me explique.
soy nuevo en todo esto btw por si hay cosas demasiado obvias que yo no vea o entienda :c 

![image](https://user-images.githubusercontent.com/99287660/160421641-5b72f3ef-97e0-47b1-8496-140e38a662b2.png)}

Aquí en la imagen se ve el código de html, sólo que no sé qué se deba poner para mostrar por la página, reitero que según, ya debería de mostrarse por el fetch pero no lo hace.

**Reply by enrique3240** [reply]
No te preocupes en lo de ser nuevo, nadie nace sabiendo. Con respecto a tu consulta, para que se muestre en la página, debes crear un elemento que utilice la información, por ejemplo un `span` con el `id="pokemonAbilities"` para ir en concordancia con la respuesta anterior de JS:

![code2](https://user-images.githubusercontent.com/17033541/160466369-eeb91c00-8295-49eb-997e-1bc23c583e4e.jpg)

Lo colocas dentro de algún `div` para acomodarlo.

El elemento `label` es para que se muestre en la página un texto que te permite identificar de qué se trata, es una etiqueta. En tu página quedaría similar a esto, antes de realizar la consulta:

![code3](https://user-images.githubusercontent.com/17033541/160467851-c01f9556-41e2-4f3c-a2bd-283182f254ed.jpg)

Y una vez que se haga la consulta te mostrará algo así:

![code4](https://user-images.githubusercontent.com/17033541/160467919-654b59be-6311-4607-8af3-aa438cb602fa.jpg)

Con ese procedimiento ya te debería aparecer en tu página las habilidades de la consulta.

**Reply by FernandoRizo** [reply]
Ahhh ya, muchísimas gracias Enrique, :D
